src/ScrollBox/index.jsx

```diff
@@ -1,509 +1,466 @@
 /*
- * Copyright (c) 2017-2019 dunnhumby Germany GmbH.
+ * Copyright (c) 2019 dunnhumby Germany GmbH.
  * All rights reserved.
  *
  * This source code is licensed under the MIT license found in the LICENSE file
  * in the root directory of this source tree.
  *
  */
 
-import React                     from 'react';
+import React, {
+    useCallback,
+    useEffect,
+    useRef,
+    useState,
+}                                from 'react';
 import PropTypes                 from 'prop-types';
-import { isEqual }               from 'lodash';
+import isEqual                   from 'lodash.isequal';
```

Comment:
`lodash.isequal` dependency was removed (since we added the whole of lodash for the date picker). So we should import `isEqual` from `'lodash'` instead